packages/shared/components/popups/AuthenticationRequiredPopup.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="typescript">
+    import { Button, TextInput, Text, TextType, HTMLButtonType } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { closePopup } from '@auxiliary/popup'
+    import { handleError } from '@core/error/handlers'
+    import type { Auth } from '@iota/wallet'
+
+    export let onSubmit: (auth: Auth) => unknown = () => {}
+
+    let username: string
+    let password: string
+    let jwt: string
+
+    let usernameError: string
+    let passwordError: string
+    let jwtError: string
+
+    $: disabled = !username && !password && !jwt
+
+    async function handleSubmit(): Promise<void> {
+        try {
+            const auth = { username, password, jwt }
+            await onSubmit(auth)
+        } catch (err) {
+            const authenticationError = err?.error?.match(/(username)|(password)|(jwt)/g)?.[0]
```

Comment:
All errors should show up at once for optimal UX.

Reply:
Hm, but I'm not sure if the error message returned from the node contains all the fields in one returned error